cobbler/tftpgen.py

```diff
@@ -356,7 +356,12 @@ def get_menu_items(self, arch: Optional[str] = None) -> dict:
         :returns: A dictionary with the pxe, ipxe and grub menu items. It has the keys from
                   utils.get_supported_system_boot_loaders().
         """
-        return self.get_menu_level(None, arch)
```

Comment:
That is not required. Please revert that. Everyone debugs differently. If you want it in then we will create an entry in the Developer Wiki on GH so everyone can find it but the code is not a Wiki itself.